File: code_blocks/🔌 Integrations/attribution/apple-search-ads-legacy_1.m
#import <iAd/iAd.h>
... 
- (BOOL)application:(UIApplication *)application 
  didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
  
  RCPurchases.automaticAppleSearchAdsAttributionCollection = YES;
  [RCPurchases configureWithAPIKey:@"public_sdk_key" appUserID:@"my_app_user_id"];

}

File: code_blocks/🔌 Integrations/attribution/apple-search-ads-legacy_1.swift
import iAd
...
func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
		
    Purchases.automaticAppleSearchAdsAttributionCollection = true
    Purchases.configure(withAPIKey: "public_sdk_key", appUserID: "my_app_user_id")
    
    return true
}